readme.md

```diff
@@ -516,7 +516,7 @@ something about it.
 When alex launched, it got some traction on [twitter][] and [producthunt][].
 Then there was a [lot][tnw] [of][dailydot] [press][vice] [coverage][bustle].
 
-There there was some backlash 🤷
+Then there was some backlash 🤷
```

Comment:
```suggestion
```

Would recommend just removing this sentence 👍

Reply:
I can’t change your fork, can you submit that suggestion?